<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fota</class>
 <widget class="QMainWindow" name="fota">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>596</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{

background:rgb(216, 216, 216)

}</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>571</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabWidget{


}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Server</string>
     </attribute>
     <widget class="QPushButton" name="Server_pushButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>171</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Server Website</string>
      </property>
     </widget>
     <widget class="QLabel" name="ShowSSID_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Your SSID</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="ShowSSID_textEdit">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
color: Yellow;

}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Path_lineEdit">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>431</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>260</y>
        <width>256</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="GetFeatures_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>171</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Get Features</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Features</string>
     </attribute>
     <widget class="QPushButton" name="LED_Button">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Led On</string>
      </property>
     </widget>
     <widget class="QPushButton" name="FingerPrint_Button">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Finger Print Register </string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Led Options</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>FingerPrint Options</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Led_Off_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>91</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Led Off</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Access_Finger_button">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>201</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Acess with Finger Print</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>511</width>
        <height>201</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>FingerPrint_Button</zorder>
     <zorder>LED_Button</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>Led_Off_button</zorder>
     <zorder>Access_Finger_button</zorder>
     <zorder>label_4</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>-10</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>FOTA Application</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>596</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
